src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
```

Comment:
you could use here proper name of variables

Reply:
Done)

Reply:
Not done, check the p3 of the checklist